src/server/common/connectors/notify/notify.js

```diff
@@ -27,31 +28,28 @@ export async function sendNotification(data) {
   let response
 
   try {
-    response = await proxyFetch(NOTIFY_URL, {
-      method: 'POST',
-      body,
+    response = await Wreck.post(NOTIFY_URL, {
+      payload,
       headers: {
         Authorization: 'Bearer ' + createToken(notifyConfig.apiKey)
       },
-      signal: AbortSignal.timeout(timeout)
+      timeout
     })
   } catch (err) {
-    if (err.code && err.code === err.TIMEOUT_ERR) {
+    if (err.output?.statusCode === statusCodes.gatewayTimeout) {
```

Comment:
It's a bit suspect to me that this works, if I'm 100% honest.  A gateway timeout is delivered to the client from a server that's acting as a proxy: https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Status/504

it's not what I'd expect to see from a HTTP client that has chosen to time _itself_ out. :thinking:

Reply:
And yet this is clearly [what Wreck is doing](https://github.com/hapijs/wreck/blob/ba28b0420d6b0998cd8e61be7f3f8822129c88fe/lib/index.js#L309)...
![image](https://github.com/user-attachments/assets/7c09af0e-acb7-49b1-bb47-31c56d091ae8)